solutions/devsprint-bruno-ramos-3/DeliveryApp/Screens/AddressSearch/AddressListView.swift

```diff
@@ -7,6 +7,103 @@
 
 import UIKit
 
-class AddressListView: UIView {
+
+// MARK: Protocols
+
+protocol AddressListViewDelegate: AnyObject {
+    func didTapOnSomeButton()
+}
+
+protocol AddressListViewProtocol: UIView {
+    var delegate: AddressListViewDelegate? { get set }
+    func display(viewModel: [AddressListView.ViewModel])
+}
+
+
+final class AddressListView: UIView, AddressListViewProtocol {
+    
+    // MARK: Properts
+    weak var delegate: AddressListViewDelegate?
+    
+    struct ViewModel {
+        let address: String
+        let name: String
+    }
+    
+    private let addressListCellIdentifier = "addressListCellIdentifier"
+    private var addressList : [ViewModel] = [] {
```

Comment:
Prefira manter o nome da su variável de acordo com o contexto para melhor legibilidade e manter o padrão
```suggestion
    private var viewModel : [ViewModel] = [] {
```